### ISSUE TITLE
Hard-coded virtual env name

### ISSUE THREAD

**Issue by whalesalad** [Bogdanp/dramatiq#2]
I am not quite sure what is happening, but I am guessing this is due to something in the build/release process?

When I run the `dramatiq-gevent` binary, I get the following error:

> dramatiq-gevent
zsh: /Users/michael/Envs/planet-ingest/bin/dramatiq-gevent: bad interpreter: /Users/bogdan/.virtualenvs/dramatiq/bin/python: no such file or directory

**Comment by Bogdanp** [comment]
Thanks for the report! It turns out that setuptools rewrites this shebang:

    #!/usr/bin/env python

To

    #!$VENV_ROOT/dramatiq/bin/python

Once I merge Ben's change, this'll be fixed.

**Comment by rakanalh** [comment]
It seems like 0.11.0 release still has the bug. Could you please mark a new release of the newer fixed version?

**Comment by rakanalh** [comment]
I checked the release on github and it's fixed... however, pypi still has 0.11.0 which has the hard coded version:
```
➜  Downloads cd dramatiq-0.11.0-py3-none-any
➜  dramatiq-0.11.0-py3-none-any ls -la
total 16
drwx------@  6 rakan  staff   204B Nov 13 22:32 .
drwx------+ 57 rakan  staff   1.9K Nov 13 22:32 ..
-rw-r--r--@  1 rakan  staff   6.0K Nov 13 22:32 .DS_Store
drwxr-xr-x@ 14 rakan  staff   476B Nov 13 22:32 dramatiq
drwxr-xr-x@  3 rakan  staff   102B Nov 13 22:32 dramatiq-0.11.0.data
drwxr-xr-x@  9 rakan  staff   306B Nov 13 22:32 dramatiq-0.11.0.dist-info
➜  dramatiq-0.11.0-py3-none-any grep -irn "bogdan" ./
.//dramatiq-0.11.0.data/scripts/dramatiq-gevent:1:#!/Users/bogdan/.virtualenvs/dramatiq/bin/python
```

**Comment by Bogdanp** [comment]
Thanks @rakanalh. It looks like this happens when I build the wheels for distribution. I thought it had been fixed but it only seemed so because I _do_ have a virtualenv called `dramatiq` :P.

Building a new wheel and then unzipping it does show the problem persists so I'm reopening this issue.

```
(dramatiq) ~/s/d/dist (master)> head dramatiq-0.11.0.data/scripts/dramatiq-gevent
#!/Users/bogdan/.virtualenvs/dramatiq/bin/python
```

**Comment by Bogdanp** [comment]
Alright, this is fixed in `0.12`:

```
(tempenv-604b214375fe0) ~> head (which dramatiq-gevent)
#!/Users/bogdan/.virtualenvs/tempenv-604b214375fe0/bin/python3.6
```

**Comment by rakanalh** [comment]
Confirmed... thanks
